### ISSUE TITLE
Nginx starts configured incorrectly for virtualhost app naming

### ISSUE THREAD

**Issue by AndyBarron** [dokku/dokku#2140]
I spent two whole days with a similar issue to this one using Dokku (0.5.4) on Digital Ocean:
https://www.digitalocean.com/community/questions/dokku-app-showing-on-all-subdomains

(Namely, deploying a single app would cause all subdomains and the root domain to point to that same app.) The issue was fixed by this command that I found in [a comment on that thread](https://www.digitalocean.com/community/questions/dokku-app-showing-on-all-subdomains?comment=31731):

````
ln -s /etc/nginx/sites-available/default /etc/nginx/sites-enabled/
service nginx reload
````

Why doesn't the app subdomain naming work out of the box? Was something misconfigured by me or Digital Ocean? This isn't mentioned in the docs, so it doesn't seem like intended behavior.

(Sincerest apologies if I have misread this and it is an issue with Digital Ocean; let me know if that's the case and I will take it to them.)

**Comment by u2mejc** [comment]
Hi @AndyBarron, I believe you're describing a known feature. The feature being that the first app deployed will respond to any and all requests to the host that have not been mapped to another deployed app, correct?

This is documented here: http://dokku.viewdocs.io/dokku/deployment/domain-configuration/#default-site

*Personally* I prefer 444 (nginx specific feature) close connection to protect your host against DNS poisoning, but that does make debugging harder, which I believe is why we still suggest to use 410 instead.

Let me know if that resolves your issue.

**Comment by AndyBarron** [comment]
Ah... That should be linked to from the quick-start docs, perhaps :)

I actually had an issue where I pushed two sites to two different subdomains, `node1` and `node2`, and all requests (from root and `node2`) were directing to `node1`. Those docs definitely explain the root behavior, but not the failure of the second app. Although thinking back, I'm pretty sure I had screwed up the DNS settings at that point...

Anyway, I'll close this out for now; I think that explains it. Thanks!!!
